lib/matplotlib/tests/test_legend.py

```diff
@@ -17,7 +17,7 @@
 import matplotlib as mpl
 import matplotlib.patches as mpatches
 import matplotlib.transforms as mtrans
-
+import matplotlib.collections as mc
```

Comment:
It's not very consistent, but this is generally imported as `mcoll` or `mcollections`. Also, second blank line should be re-added.